go/updater/updater.go

```diff
@@ -307,7 +310,10 @@ func (u *Updater) apply(src string, dest string) (tmpPath string, err error) {
 
 // Update checks, downloads and performs an update.
 func (u *Updater) Update(ui libkb.UpdateUI, force bool, requested bool) (update *keybase1.Update, err error) {
-	update, err = u.CheckForUpdate(false, force, requested)
+	u.Lock()
+	defer u.Unlock()
```

Comment:
Thanks, I kind of understand.  What do you mean by time check? Can you show me where in the code that is? Thanks